docs/5.0/application-access.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Application Access Guide
+description: How to set up and configure Teleport for Application access with SSO and RBAC
+---
+
+# Teleport Application Access
+
+## Introduction
+Application Access has been designed to provide secure access to internal dashboards and applications. Building on Teleports strong foundations of security and identity. You can now put applications onto the internet safely and securely.
+
+Here are a few things you might want to secure with Teleport Application Access
+
+* Internal Control Panels
+* Wikis / Tooling that's only available on the VPN.
+* Access to the Kubernetes Dashboard
+* Developer tools. Such as Jenkins, or the Atlassian stack.
+
+#### Hardware Requirements
+We recommend reviewing our [How it works](https://gravitational.com/teleport/how-it-works/) page to get a good overview of how Teleport works. You'll need a small VM or even a Raspberry Pi to run Teleport (Auth and Proxy).  This will be the brains and gateway to your applications.
+
+#### Networking Requirements
+
+!!! note "Why do I see SSH Ports for Teleport Application Access?"
+
+    Below is a list of ports required for Teleport Application Access to work but why SSH ports? The reason for this is Teleport uses reverse tunnels to obtain access to these applications.
```

Comment:
Teleport uses SSH reverse tunnels to connect applications to the proxy. This is why configuration below mentions SSH ports.